packages/api/cms-api/src/cron-jobs/cron-jobs.resolver.ts

```diff
@@ -3,9 +3,11 @@ import { Query, Resolver } from "@nestjs/graphql";
 import { BUILDER_LABEL } from "../builds/builds.constants";
 import { INSTANCE_LABEL } from "../kubernetes/kubernetes.constants";
 import { KubernetesService } from "../kubernetes/kubernetes.service";
+import { RequiredPermission } from "../user-permissions/decorators/required-permission.decorator";
 import { CronJob } from "./dto/cron-job.object";
 
 @Resolver(() => CronJob)
+@RequiredPermission(["pageTree"], { skipScopeCheck: true })
```

Comment:
This should probably be a separate permission (e.g. "system")

Reply:
https://github.com/vivid-planet/comet/pull/1223/commits/e1863973d977aa96544ce4823544252330a48bb6